bindings/python/opendaq/generated/component/py_component_private.cpp

```diff
@@ -44,35 +44,35 @@ void defineIComponentPrivate(pybind11::module_ m, PyDaqIntf<daq::IComponentPriva
             objectPtr.lockAttributes(attributes);
         },
         py::arg("attributes"),
-        "");
+        "Adds the attributes contained in the list to the set of locked components.");
     cls.def("lock_all_attributes",
         [](daq::IComponentPrivate *object)
         {
             const auto objectPtr = daq::ComponentPrivatePtr::Borrow(object);
             objectPtr.lockAllAttributes();
         },
-        "");
+        "Locks all attributes of the component.");
     cls.def("unlock_attributes",
         [](daq::IComponentPrivate *object, daq::IList* attributes)
         {
             const auto objectPtr = daq::ComponentPrivatePtr::Borrow(object);
             objectPtr.unlockAttributes(attributes);
         },
         py::arg("attributes"),
-        "");
+        "Removes the attributes contained in the list from the set of locked components.");
```

Comment:
i see that another functions has description like lock/unlock attrubutes and this one has description which from the first look seems different. maybe rename it like "unlock set of attributes of the component"?